lib/license.js

```diff
@@ -50,18 +94,18 @@ function fixCli(pkgPath) {
     switch (license.name) {
       case 'StrongLoop': // OBSOLETE
         file.data.license = 'SEE LICENSE IN LICENSE.md';
-        writeStrongloopLicense();
+        license = writeStrongloopLicense();
```

Comment:
Yeah, its not very nice ATM. The whole file would be better rewritten as a "current license auto-guess" function, and a "license set" function... but I don't have the time to spend on this - making this nice falls into the YGNI as far as I'm concerned. And if I do need it, I can fix it then.